### PR TITLE
add adWMGAnalyticsAdapter

### PR DESCRIPTION
**Pull request by nyakove**
adWMG Analytics is a module for collecting dynamic data and analytics acquisition developed by WMG. It enables compiling and saving of the auction history, its results, users’ data (geo, browser, operation system). With the access to the platform, the customer may explore the product in a preferable way.

## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Adding adWMG Analytics Adapter

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/1988

File: modules/adWMGAnalyticsAdapter.js [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+let bidTimeOutArray = [];
+let bidTimeOutObject = {};
+
+let bidWonArray = [];
+let bidWonObject = {};
+
+let initOptions = {};
+
+function postAjax(url, data) {
+  var xhr = new XMLHttpRequest();
```
Comment (Fawke) [inline modules/adWMGAnalyticsAdapter.js:35]: Please use Prebid inbuilt ajaxBuilder found in `src/ajax.js`

**Comment by nyakove** [comment]
Thanks for the review, @Fawke. I'll make changes and update the code soon

**Comment by nyakove** [comment]
Hi @Fawke! I've just update a code, please take a look!

**Pull request merged**